specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -8842,6 +8924,114 @@
         "modelAsString": true
       }
     },
+    "FlowAccessControlConfiguration": {
+      "type": "object",
+      "description": "The access control configuration.",
+      "properties": {
+        "triggers": {
+          "$ref": "#/definitions/FlowAccessControlConfigurationPolicy",
+          "description": "The run workflow triggers."
+        },
+        "contents": {
+          "$ref": "#/definitions/FlowAccessControlConfigurationPolicy",
+          "description": "The run workflow contents."
+        },
+        "actions": {
+          "$ref": "#/definitions/FlowAccessControlConfigurationPolicy",
+          "description": "The run workflow actions."
+        },
+        "workflowManagement": {
+          "$ref": "#/definitions/FlowAccessControlConfigurationPolicy",
+          "description": "The workflow management."
+        }
+      }
+    },
+    "FlowAccessControlConfigurationPolicy": {
+      "type": "object",
+      "description": "The access control configuration policy.",
+      "properties": {
+        "allowedCallerIpAddresses": {
+          "type": "array",
+          "description": "The allowed caller IP address ranges.",
+          "items": {
+            "$ref": "#/definitions/IpAddressRange",
+            "description": "The ip address range."
+          }
+        },
+        "openAuthenticationPolicies": {
+          "$ref": "#/definitions/OpenAuthenticationAccessPolicies",
+          "description": "The authentication policies for flow."
```

Comment:
The authentication policies for workflow.

Reply:
resolved in latest commit.